Ml commons (#5017)
opensearch-project/documentation-website: _ml-commons-plugin/index.md

Should you not want to use a model, you can use the [Train and Predict]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#train-and-predict) API to test your model without having to evaluate the model's performance.

> ## Utilizing ML Commons

Reviewer: ```suggestion
## Using ML Commons
```

> 1. Ensure you've set the appropriate cluster settings outlined in [Cluster Settings]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings/). 

Reviewer 2: ```suggestion
1. Ensure that you've appropriately set the cluster settings described in [Cluster Settings]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings/). 
```

> 2. Set up model access described in [Model Access Control]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control/). 

Reviewer 2: ```suggestion
2. Set up model access as described in [Model Access Control]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control/). 
```

> 3. Start utilizing models: 

Reviewer: ```suggestion
3. Start using models: 
```

>   - [ML Framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/ml-framework/) for running models within OpenSearch. 

Reviewer: ```suggestion
  - [ML Framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/ml-framework/) allows you to run models within OpenSearch. 
```

>   - [ML Extensibility]({{site.url}}{{site.baseurl}}/ml-commons-plugin/extensibility/index/) to access remote models. 

Reviewer: ```suggestion
  - [ML Extensibility]({{site.url}}{{site.baseurl}}/ml-commons-plugin/extensibility/index/) allows you to access remote models. 
```
